foqus_lib/framework/optimizer/BFGS.py

```diff
@@ -1,3 +1,18 @@
+###############################################################################
+# FOQUS Copyright (c) 2012 - 2021, by the software owners: Oak Ridge Institute
+# for Science and Education (ORISE), TRIAD National Security, LLC., Lawrence
+# Livermore National Security, LLC., The Regents of the University of
+# California, through Lawrence Berkeley National Laboratory, Battelle Memorial
+# Institute, Pacific Northwest Division through Pacific Northwest National
+# Laboratory, Carnegie Mellon University, West Virginia University, Boston
+# University, the Trustees of Princeton University, The University of Texas at
+# Austin, URS Energy & Construction, Inc., et al.  All rights reserved.
+#
+# Please see the file LICENSE.md for full copyright and license information,
+# respectively. This file is also available online at the URL
+# "https://github.com/CCSI-Toolset/FOQUS".
+#
+###############################################################################
 """#FOQUS_OPT_PLUGIN BFGS.py
 
 Optimization plugins need to have #FOQUS_OPT_PLUGIN in the first 150 characters,
```

Comment:
The failing tests are because of a `KeyError` with `BFGS` not being in the `optPlugins` dictionary of the `session`/`self.dat` instance. Maybe this requirement to identify plugins was inadvertently broken by the added header, causing the BFGS plugin to be ignored, and in turn the tests to fail?

Reply:
@lbianchi-lbl I am looking into this, and will follow up soon.